components/eam/src/dynamics/se/inidat.F90

```diff
@@ -47,7 +47,10 @@ subroutine read_inidat( ncid_ini, ncid_topo, dyn_in)
     use chemistry,               only: chem_implements_cnst, chem_init_cnst
     use tracers,                 only: tracers_implements_cnst, tracers_init_cnst
     use aoa_tracers,             only: aoa_tracers_implements_cnst, aoa_tracers_init_cnst
-    use cldera_sai_tracers,      only: cldera_sai_tracers_implements_cnst, cldera_sai_tracers_init_cnst
+    use cldera_sai_tracers,      only: cldera_sai_tracers_implements_cnst, &
+                                       cldera_sai_tracers_init_cnst
+    use cldera_passive_tracers,    only: cldera_passive_tracers_implements_cnst, &
```

Comment:
May as well line up the `only`’s here with the lines above and below for consistency.